templates/author.twig

```diff
@@ -0,0 +1,72 @@
+{% extends "base.twig" %}
+
+{% block content %}
+    <header class="page-header container">
+        {% if ( author.avatar ) %}
+            <figure class="author-image" aria-label="{{ __( 'Picture of ' ~ author.name , 'planet4-master-theme' ) }}">
+                <img itemprop="image" class="author-pic"
+                     src="{{ fn('get_avatar_url', author.id, {'size' : 300}) | replace({'=s96-c':""}) }}"
+                     alt="{{ author.name }}">
+            </figure>
+        {% endif %}
+        <div class="author-details">
+            <h1 class="page-header-title">{{ author.name }}</h1>
+
+            {% if role %}
+                <h2>{{ role|raw }}</h2>
+            {% endif %}
+
+            {% if role %}
+                <p class="bio-linkedin"><a href="{{ linkedinProfile|raw }}" target="_blank">Find me on LinkedIn</a></p>
+            {% endif %}
+
+            {% if ( author.description ) %}
+                <p class="d-md-block author-bio" aria-label="{{ __( 'Author bio', 'planet4-master-theme' ) }}">{{ fn('wpautop', fn('the_author_meta', 'description', author.id))|e('wp_kses_post')|raw }}</p>
```

Comment:
I think we need to look closesly at this line and figure out what of it we need to replicate in our custom additions

for example, `wp_kses_post` does some security filtering and so does "e|" (short for "escape|").